librz/bin/dwarf.c

```diff
@@ -2060,10 +2060,10 @@ static ut8 *get_section_bytes(RzBinFile *binfile, const char *sect_name, size_t
 	if (!section) {
 		return NULL;
 	}
-	if (section->size > binfile->size) {
+	if (section->paddr >= binfile->size) {
```

Comment:
shouldn't we also check the ->size of the section?

Reply:
@ret2libc Check is done here, so we get either the size of the section or from the section start until the end of the file  https://github.com/rizinorg/rizin/blob/0bc9779e018a4e549ec791be816260b7e6684314/librz/bin/dwarf.c#L2066